cmd/nvidia-ctk/cdi/generate/generate.go

```diff
@@ -169,6 +174,11 @@ func (m command) build() *cli.Command {
 			Usage:       "Specify a pattern the CSV mount specifications.",
 			Destination: &opts.csv.ignorePatterns,
 		},
+		&cli.BoolFlag{
+			Name:        "--allow-additional-gids",
+			Usage:       "Allow the use of the additionalGIDs field for generated CDI specifications. Note this will generate a v0.7.0 CDI specification.",
```

Comment:
Which versions of containerd / cri-o (and other runtimes for that matter) support this?

Reply:
I would have to check. Note that this is not enabled by default when generating the CDI spec and only for internal representations.

Reply:
* containerd (unreleased) - https://github.com/containerd/containerd/commit/7a2f49f70f1d2cacfededbbf65452d481476bc10
* docker (>= 26.1.0) - https://github.com/moby/moby/commit/745e2356ab01363a02cfec947c624a9f6220fe7f
* podman (>= 5.1.0) - https://github.com/containers/podman/commit/a40cf3195acb6ac5fea5ab4617afb99006a3bed7
* crio (>= 1.30.0) - https://github.com/cri-o/cri-o/commit/fd9aa76250fe05625d8c968b922cd1a0ae88eb1b